### ISSUE TITLE
Questions on layer1&2 's dropblock

### ISSUE THREAD

**Issue by tao0420** [kjunelee/MetaOptNet#28]
Hi there,

Thanks for the work!

I got a bit confused for some details of dropblocks. Please correct me if I misunderstand something.
As mentioned in the paper, both for ImageNet and CIFAR dataset, the layer1 and layer2's setting of dropblock is DB(0.9, 1) which means that in the code the parameters should be drop_rate=0.1, drop_size=1;

However when I looked through the code, the initialization of layer1 and layer2 serves as:
self.layer1 = self._make_layer(block, 64, stride=2, drop_rate=drop_rate);
Although the default drop_size=1, the default drop_block=False, which means that if initialized as this, with the code:
        if self.drop_rate > 0:
            if self.drop_block == True:
                feat_size = out.size()[2]
                keep_rate = max(1.0 - self.drop_rate / (20 * 2000) * (self.num_batches_tracked), 1.0 - self.drop_rate)
                gamma = (1 - keep_rate) / self.block_size ** 2 * feat_size ** 2 / (feat_size - self.block_size + 1) ** 2
                out = self.DropBlock(out, gamma=gamma)
            else:
                out = F.dropout(out, p=self.drop_rate, training=self.training, inplace=True)

It is actually used the dropout rather than dropblock. I am wondering if this is correct and it is consistent with what mentioned in the paper?

Thanks for your time!

**Comment by kjunelee** [comment]
This is a great question!

It is because the DropBlock paper says "DropBlock resembles dropout [1] when block_size = 1"
